actors/miner/src/types.rs

```diff
@@ -127,6 +127,39 @@ pub struct ProveCommitSectorParams {
     pub proof: RawBytes,
 }
 
+// Note no UnsealedCID because it must be "zero" data.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize_tuple, Deserialize_tuple)]
+pub struct SectorNIActivationInfo {
+    pub sealing_number: SectorNumber,
+    pub sealer_id: ActorID,
+    pub sealed_cid: Cid, // CommR
+    pub sector_number: SectorNumber,
+    pub seal_rand_epoch: ChainEpoch,
+    pub expiration: ChainEpoch,
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize_tuple, Deserialize_tuple)]
+pub struct ProveCommitSectorsNIParams {
+    // Information about sealing of each sector.
+    pub sectors: Vec<SectorNIActivationInfo>,
+
+    // Aggregate proof for all sectors.
+    // Exactly one of sector_proofs or aggregate_proof must be non-empty.
+    pub aggregate_proof: RawBytes,
+
+    // Proof type for each seal (must be an NI-PoRep variant)
+    pub seal_proof_type: RegisteredSealProof,
+
+    // Proof type for aggregation, if aggregated
+    pub aggregate_proof_type: RegisteredAggregateProof,
+
+    // The Window PoST deadline index at which to schedule the new sectors
+    pub proving_deadline: u64,
+
+    // Whether to abort if any sector activation fails.
+    pub require_activation_success: bool,
```

Comment:
```suggestion
    pub sectors: Vec<SectorNIActivationInfo>, // Information about sealing of each sector
    pub aggregate_proof: RawBytes,            // Aggregate proof for all sectors
    pub seal_proof_type: RegisteredSealProof, // Proof type for each seal (must be an NI-PoRep variant)
    pub aggregate_proof_type: RegisteredAggregateProof, // Proof type for aggregation
    pub proving_deadline: u64, // The Window PoST deadline index at which to schedule the new sectors
    pub require_activation_success: bool, // Whether to abort if any sector activation fails
```

a couple of the comments are off now with batch, this updates it according to the FIP but they also are short enough to be on the same line now